source/getting-started/setup.rst

```diff
@@ -76,7 +76,7 @@ Netzwerkmaske ``255.255.0.0``, dem Gateway ``10.0.0.254`` und dem DNS
 ``10.0.0.1``.
 
 Öffne auf dem Admin-PC mit einem Webbrowser die URL
-``http://10.0.0.1``. Melde dich hier einmalig mit dem Benutzer
+``http://10.0.0.1:8000``. Melde dich hier einmalig mit dem Benutzer
```

Comment:
Bei der manuellen Installation ist die WebUI unter Port 80 erreichbar.

Reply:
danke für die schnelle Rückmeldung, ists dann besser man erwähnt hier die 2 Wege? Ich hatte aktuell nur die esxi images probiert....

Reply:
Besser wäre es man würde der Ursache auf den Grund gehen. Ich meine in Erinnerung zu haben, dass Port 80 jetzt der Standardport ist. Vielleicht sind die Images noch nicht aktualisiert? Am besten du fragst mal im Forum nach (ask.linuxmuster.net) und schilderst dort das Problem.

Reply:
Was soll mit dem PR passieren, da mittlerweile die Ursache behoben ist.

[https://ask.linuxmuster.net/t/esxi-images-von-netzint-initial-port-8000/5534/7](https://ask.linuxmuster.net/t/esxi-images-von-netzint-initial-port-8000/5534/7)